app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Like < ApplicationRecord
+  belongs_to :post_like, foreign_key: 'post_id', class_name: 'Post'
+  belongs_to :user_like, foreign_key: 'user_id', class_name: 'User'
```

Comment:
- [ ] **[OPTIONAL]** It's suggested to add uniqueness validation on [user_id, post_id] combination for `likes` table. You can use [`scope`](https://guides.rubyonrails.org/active_record_validations.html#uniqueness) to validate uniqueness of multiple columns.